workers/unity/Packages/io.improbable.gdk.core/.schema/improbable/gdk/editor/worker_inspector.schema

```diff
@@ -0,0 +1,5 @@
+package improbable.gdk.editor;
+
+type ComponentIcon {
+    string icon_name = 1;
+}
```

Comment:
missing eol, is the editorconfig file not setup for schema?

Reply:
It is but you have to open it in an editor which supports `editorconfig` files, VSCode apparently, does not out of the box.